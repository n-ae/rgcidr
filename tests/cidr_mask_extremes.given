0.0.0.0
0.0.0.1
127.255.255.255
128.0.0.0
255.255.255.254
255.255.255.255
192.168.0.0
192.168.0.1
192.168.0.127
192.168.0.128
192.168.0.255
192.168.1.0
::
::1
ffff:ffff:ffff:ffff:ffff:ffff:ffff:fffe
ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff
8000::
7fff:ffff:ffff:ffff:ffff:ffff:ffff:ffff
2001:db8::
2001:db8:0000:0000:0000:0000:0000:0001
