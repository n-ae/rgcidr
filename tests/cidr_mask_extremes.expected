0.0.0.0
0.0.0.1
127.255.255.255
128.0.0.0
255.255.255.254
255.255.255.255
192.168.0.0
192.168.0.1
192.168.0.127
192.168.0.128
192.168.0.255
192.168.1.0
